Github/service.js

```diff
@@ -14,9 +14,9 @@ const key = process.env.GITHUB_TOKEN;
 const repoUrl = `https://api.github.com/orgs/Seneca-CDOT/repos?per_page=100&access_token=${key}`;
```

Comment:
The url parameter in each `request.get(...)` call looks quite lengthy, I am thinking we could store the base url for the github service as `const baseUrl=https://api.github.com` and reuse it to make it more readable.

Alternatively, `request` supports default options https://www.npmjs.com/package/request#requestdefaultsoptions  & you could set `baseUrl` in there